lib/KThemePlugin.js

```diff
@@ -39,13 +40,34 @@ import KCard from './KCard';
 import { themeTokens, themeBrand, themePalette, themeOutlineStyle } from './styles/theme';
 import globalThemeState from './styles/globalThemeState';
 
+import useKLiveRegion from './composables/useKLiveRegion';
+
+const { _mountLiveRegion } = useKLiveRegion();
+
 require('./grids/globalStyles.js'); // global grid styles
 
 /**
  * Install Kolibri theme helpers on all Vue instances.
  * Also, set up global state, listeners, and styles.
  */
 export default function KThemePlugin(Vue) {
+  // Note that if DOM live regions need to be demostrated
+  // on the KDS website, and therefore attached to the DOM,
+  // just call _mountLiveRegion() in the relevant documentation
+  // page's 'mounted' (see 'docs/pages/usekliveregio.vue' for an example)
+  if (!isNuxtServerSideRendering()) {
+    const onDomReady = () => {
+      _mountLiveRegion();
+      document.removeEventListener('DOMContentLoaded', onDomReady);
+    };
+
+    if (document.readyState === 'loading') {
+      document.addEventListener('DOMContentLoaded', onDomReady);
```

Comment:
The only issue would be is if this is executing in the <head> tag of the page, and so the body element doesn't exist yet.

In Kolibri, this would work, but in other contexts it might not depending on when execution occurs.

Reply:
Oooh, thats true, alright then, thank you!

Reply:
Yes, the point is not only to insert it before Vue is loaded, but also after HTML is ready.

Besides, `DOMContentLoaded` listener will be added only if `document.readyState === 'loading'`. If the document is not loading, then live regions will be inserted right away as you suggest @AlexVelezLl.

Perhaps we won't run into it, but it's at least a polite thing to do that doesn't harm anything and increases robustness a bit.

Reply:
Yees, I agree. Thank you!